shared/messaging/utils.go

```diff
@@ -0,0 +1,37 @@
+package messaging
+
+import (
+	"log"
+
+	"google.golang.org/protobuf/proto"
+	"google.golang.org/protobuf/reflect/protoreflect"
+	"google.golang.org/protobuf/types/known/anypb"
+)
+
+// PackMessage returns a *PocketEnvelope after having packed the message supplied as an argument
+func PackMessage(message proto.Message) (*PocketEnvelope, error) {
+	anyMsg, err := anypb.New(message)
+	if err != nil {
+		return nil, err
+	}
+	return &PocketEnvelope{Content: anyMsg}, nil
+}
+
+// MustPackMessage returns a *PocketEnvelope after having packed the message supplied as an argument and fails fatally in case of errors
+func MustPackMessage(message proto.Message) *PocketEnvelope {
```

Comment:
I'm not convinced we should have `Must` functions in our codebase. `log.Fatalf` or `panic` in non-debug code in blockchains can lead to Validators crashing or exploits. I encourage us to remove these functions if possible. 

If we decide to leave it. Then it should have a counterpart `MustUnpackMessage`